cob_monitoring/src/netdata_tools/netdata_tools.py

```diff
@@ -0,0 +1,45 @@
+import rospy
+import requests
+import numpy
+
+def query_netdata_info():
+
+    try:
+        r = requests.get('http://127.0.0.1:19999/api/v1/info')
+    except requests.ConnectionError as ex:
+        rospy.logerr("NetData ConnectionError %r", ex)
+        return None
```

Comment:
maybe we should return a tuple (data, error) - so if data is None we can write error into the diagnostics message

this should be done for all error cases in this `netdata_tools` module